helpers/calendar/analytics_helper.py

```diff
@@ -224,3 +225,25 @@ def get_list_of_month_dates(start_date, start_dt, end_date, end_dt):
         last_month_start_date = datetime.strptime(last_month_start, '%b %d %Y').isoformat() + 'Z'  # noqa E501
         dates.append([last_month_start_date, end_date])
         return dates
+
+    @staticmethod
+    def get_bookings_count_from_events(start_date, end_date, events, room_id):
+        start_dt = dateutil.parser.parse(start_date)
```

Comment:
@rajeman Avoid the use of shortened/abbreviated variable names. Also, what's `i` from line 35? Use variable names that are descriptive.

Reply:
Thanks @ThaDeveloper. Changes have been made